acceptance/analyzer_test.go

```diff
@@ -499,6 +575,16 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 			when("app image exists", func() {
 				it("does not restore app metadata", func() {
 					h.SkipIf(t, api.MustParse(platformAPI).Compare(api.MustParse("0.7")) < 0, "Platform API < 0.7 restores app metadata")
+
+					analyzeFlags := []string{"-daemon"}
+					if api.MustParse(platformAPI).Compare(api.MustParse("0.7")) >= 0 {
+						analyzeFlags = append(analyzeFlags, []string{"-run-image", "some-run-image"}...)
```

Comment:
I think you don't need to wrap this with the if condition because we skip the test for platformAPI < 0.
So you can just initialize the flags with both the `-daemon` and the `-run-image` flags.

Reply:
Good catch!

Reply:
Updated in 07e41be